pkgs/top-level/all-packages.nix

```diff
@@ -3432,6 +3432,8 @@ with pkgs;
 
   dbus-broker = callPackage ../os-specific/linux/dbus-broker { };
 
+  dbx = python3Packages.callPackage ../development/python-modules/dbx { };
```

Comment:
Either move this into pythonPackages or move the file out of python-modules. If it is a cli that is not used as a dependency the first is highly preferred.

Reply:
> Either move this into pythonPackages or move the file out of python-modules. If it is a cli that is not used as a dependency the first is highly preferred.

Did you mean "the later" is preferred? wrt to your previous comment https://github.com/NixOS/nixpkgs/pull/163709#discussion_r825176228

Reply:
If the package has only an entry in all-packages then it should not by under python-modules. For that it needs an entry in python-packages.nix.

Reply:
I've moved it to pythonPackages, so that it's like `databrick-cli` ([ref](https://github.com/NixOS/nixpkgs/blob/nixos-22.05/pkgs/development/python-modules/databricks-cli)). Let me know if that's ok.

Reply:
Yes but I thought this was a cli which is not used as a python module. 

I would suggest to move it under pkgs/applications/misc